standalone/src/test/java/io/pravega/local/InProcPravegaClusterTest.java

```diff
@@ -74,32 +71,41 @@ public void createTestStream()
         String streamName = "Stream";
         int numSegments = 10;
 
-        ClientConfig clientConfig = ClientConfig.builder()
-                                                .controllerURI(URI.create(localPravega.getInProcPravegaCluster().getControllerURI()))
-                                                .credentials(new DefaultCredentials("1111_aaaa", "admin"))
-                                                .trustStore("../config/cert.pem")
-                                                .validateHostName(false)
-                                                .build();
-        @Cleanup
-        StreamManager streamManager = StreamManager.create(clientConfig);
+        try {
+            System.setProperty("pravega.client.auth.method", "Pravega-Default");
+            System.setProperty("pravega.client.auth.username", "admin");
```

Comment:
This particular test case is already present `CredentialsExtractorTest`. Do you recommend dropping these changes?

Reply:
I'm not sure I understand. If the test case already exists, why haven't it captured this problem?

Let me try to clarify it, though. My main concern is replacing:

```
.userName("admin")
.passwd("1111_aaaa")
```

with:

```
System.setProperty("pravega.client.auth.username", "admin");
System.setProperty("pravega.client.auth.password", "1111_aaaa");
```

because of side effects. I think this change is not necessary and it is sufficient to to verify that the behavior of `extractCredentialsFromProperties` and `extractCredentialsFromEnv` is as expected.

Reply:
Dropped the changes causing side effects in tests.

Reply:
The test case `testExtractCredentials` in `CredentialsExtractorTest.java` covers this part. I have not added the env part because I was not able to find a way to set an env without causing a side effect in the test.
We did not catch it earlier because this test was checking for the full string `pravega.client.auth.prop1` where as this should have cheked just for `prop1` (should have dropped the prefix). This PR fixes that test case too.